jscomp/build_tests/in_source/input.js

```diff
@@ -1,23 +1,23 @@
-var child_process = require("child_process");
-
-var assert = require("assert");
-
-var rescript_exe = require("../../../scripts/bin_path").rescript_exe;
-
-assert.throws(
-  () => {
-    var output = child_process.execSync(`${rescript_exe} build -regen`, {
-      cwd: __dirname,
-      encoding: "utf8",
-    });
-  },
-  function (err) {
-    if (err.message.match(/detected two module formats/)) {
-      return true;
-    }
-    return false;
-  }
-);
+//var child_process = require("child_process");
```

Comment:
Why is this commented out? Couldn't we keep the tests and just remove the `-regen`?

Reply:
Yes, I will keep it. I removed it just to test on CI